config/config.go

```diff
@@ -483,6 +484,30 @@ func (c *Configuration) loadGitConfig() {
 	}
 }
 
+// All the date formats that Git accepts, except for the built-in one, which is
```

Comment:
Nit: godoc comments must start with the name of the identifier, e.g., `// dateFormats is a list of all the ...`.

Reply:
Good point.  Will fix.